cli/cmd/routes.go

```diff
@@ -233,7 +234,11 @@ func printRouteTable(stats []*routeRowStats, w *tabwriter.Writer, options *route
 
 			fmt.Fprintf(w, templateString, values...)
 		} else {
-			fmt.Fprintf(w, emptyTemplateString, values...)
+			if outputActual {
+				fmt.Fprintf(w, emptyWideTemplateString, values...)
+			} else {
+				fmt.Fprintf(w, emptyTemplateString, values...)
+			}
```

Comment:
Instead of checking `outputActual` for each row in the stats, can we just check it once above and set the template string based off that?:
```
var emptyTemplateString string
if outputActual {
    emptyTemplateString = routeTemplate + "\t%s\t-\t-\t-\t-\t-\t-\t-\t\n"
} else {
    emptyTemplateString = routeTemplate + "\t%s\t-\t-\t-\t-\t-\t\n"
}
..
```

Reply:
Yep, Great suggestion. Updated. :)